src/ui/public/vis/vis.js

```diff
@@ -36,6 +46,7 @@ export default function VisFactory(Notifier, Private) {
 
     let schemas = type.schemas;
 
+    // NOTE: What is this doing?
```

Comment:
Basically, this was put in place to do migrations at runtime. I'm not certain when it went in, but I'm certain we can remove it for 5.0